bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/BridgeType.java

```diff
@@ -63,9 +65,10 @@ public BridgeType(String bindingId, String thingTypeId, String label) throws Ill
      * @throws IllegalArgumentException if the UID is null or empty,
      *             or the the meta information is null
      */
-    public BridgeType(ThingTypeUID uid, List<String> supportedBridgeTypeUIDs, String label, String description,
-            List<ChannelDefinition> channelDefinitions, List<ChannelGroupDefinition> channelGroupDefinitions,
-            Map<String, String> properties, URI configDescriptionURI) throws IllegalArgumentException {
+    public BridgeType(@NonNull ThingTypeUID uid, @Nullable List<String> supportedBridgeTypeUIDs, @NonNull String label,
```

Comment:
Didn't we agree on that we only want to state if a parameter is `NonNull` and if if can be `null` we omit the `@Nullable` annotation?